src/envoy/http/service_control/handler_impl_test.cc

```diff
@@ -194,11 +196,27 @@ class HandlerTest : public ::testing::Test {
     counter.reset();
   }
 
+  void setPerRouteOperation(const std::string& operation) {
+    ::espv2::api::envoy::v9::http::service_control::PerRouteFilterConfig
+        per_route_cfg;
+    per_route_cfg.set_operation_name(operation);
+    auto per_route = std::make_shared<PerRouteFilterConfig>(per_route_cfg);
```

Comment:
Should this be saved as a class member variable? It will be used in the lambda after this `setPerRouteOperation` function returns. I guess the lambda captures the shared_pointer, but it looks confusing.

Reply:
Yes, lambda capture add a reference to the shared_ptr.  
Not need to be a test class member since this is the only place using it.